libs/coin-framework/src/api/types.ts

```diff
@@ -11,12 +10,28 @@ export type BlockInfo = {
   parent?: BlockInfo;
 };
 
+// NOTE: from crypto-asset
+export type Unit = {
+  // display name of a given unit (example: satoshi)
+  name: string;
+  // string to use when formatting the unit. like 'BTC' or 'USD'
+  code: string;
+  // number of digits after the '.'
+  magnitude: number;
+  // should it always print all digits even if they are 0 (usually: true for fiats, false for cryptos)
+  showAllDigits?: boolean;
```

Comment:
This `showAllDigits` and `prefixCode` properties are useless here as it is only useful for UI client.
Or have I missed something?